pkg/ssh/keymanager_test.go

```diff
@@ -1,3 +1,4 @@
+//nolint:goconst,nolintlint
```

Comment:
how hard would it be to fix these lint errors instead of ignoring them?

Reply:
The linter was complaining about some strings that are not constants. The strings are being concatenated in the tests (e.g. someVar + "string"). We could move the strings to constants but I don't think it would make the code more readable.

Reply:
Created some constants and started using them to avoid having to disable the linter.